.github/workflows/open-datadog-agent-pr.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           commit-message: 'omnibus: bump OMNIBUS_SOFTWARE_VERSION'
           title: '[omnibus][automated] Bump OMNIBUS_SOFTWARE_VERSION'
           branch: 'automated/omnibus-software/${{ github.event.number }}'
-          labels: 'team/agent-platform,changelog/no-changelog,qa/no-code-change,qa/skip-qa'
```

Comment:
Could you also update the PR body on line 83? (I can't comment on it directly because it's not part of your change :( )